src/CommunityToolkit.Maui/Converters/StringToListConverter.shared.cs

```diff
@@ -11,6 +11,9 @@ public class StringToListConverter : BaseConverterOneWay<string?, IEnumerable, o
 	string separator = " ";
 	IList<string> separators = Array.Empty<string>();
 
+	/// <inheritdoc/>
+	public override IEnumerable DefaultReturnValue { get; } = Array.Empty<string>();
```

Comment:
`Array.Empty<string>()` matches the default return value in the `ConvertFrom()` method.

(snippet from `ConvertFrom()`):
```cs
if (value is null)
{
  return Array.Empty<string>();
}
```